packages/helpers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paprika/helpers",
-  "version": "0.2.4",
+  "version": "0.2.5",
```

Comment:
do i need to bump this? @nahumzs

Reply:
no only the dependency inside of the side panel. pretty much bump the version only on the sidepanel package.json

Reply:
okok my bad

Reply:
lerna will do this automatically after I describe it, the sidepanel is a breaking change. as well this one.